server/index.js

```diff
@@ -120,6 +120,48 @@ app.post('/api/auth/sign-in', (req, res, next) => {
     })
     .catch(err => next(err));
 });
+
+app.get('/api/messages/:recipientId', (req, res, next) => {
+  const { recipientId } = req.params;
```

Comment:
instead of adding a recipient id in the url path, you can plug in the authorization middleware before this route and then use the `req.user.userId`